src/index.html

```diff
@@ -12,7 +12,48 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__menu">
+    <div class="stars stars--0">
+      <div class="star__element"></div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars__element"></div>
```
follow the BEM pattern (block__element)